src/main/java/org/fundacionjala/coding/omar/Kata7.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.omar;
+
+/**
+ * This class contains a method that prints the numbers greater to 0.
+ * But for multiples of three print “Fizz” instead of the number
+ * and for the multiples of five print “Buzz”.
+ * For numbers which are multiples of both three and five print “FizzBuzz “.
+ */
+public class Kata7 {
+
+
+    private static final int NUMBER_15 = 15;
+    private static final int ZERO = 0;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_5 = 5;
+
+    /**
+     * @param number this is words for Spin Words
+     * @return result
+     */
+    public String isFizzBuzzNumer(int number) {
+
+        return number % NUMBER_15 == ZERO
+                ? "FizzBuzz" : number % NUMBER_3 == ZERO
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)